UE/Tests/Application/Ports/UserPortTestSuite.cpp

```diff
@@ -54,7 +57,33 @@ TEST_F(UserPortTestSuite, shallShowMenuOnConnected)
     EXPECT_CALL(guiMock, setListViewMode()).WillOnce(ReturnRef(listViewModeMock));
     EXPECT_CALL(listViewModeMock, clearSelectionList());
     EXPECT_CALL(listViewModeMock, addSelectionListItem(_, _)).Times(AtLeast(1));
+    EXPECT_CALL(guiMock, setAcceptCallback(_));
     objectUnderTest.showConnected();
 }
 
+TEST_F(UserPortTestSuite, shallViewSmsList)
+{
+    using SmsMessages = std::vector<std::pair<Sms, SmsState>>;
```

Comment:
redeclaration

Reply:
Where it is redeclarated? With out that SmsMessages is unknown.